mantidproject/mantid
# ZAxisEditor initialised with wrong tick format

**Description of work.**
When opening the "edit axis" dialog for the z-axis of a 3D plot, the tick format was being read from the y-axis, rather than the z-axis. This is now fixed.

**To test:**
1. Load some data and open a 3D surface or wireframe plot.
2. Open the edit axis dialog for the z-axis by double clicking a value on the axis (see screenshot below).
3. Change the  `Tick Format` to `Scientific Format`.
4. Close the dialog.
5. Open it again - the combo box should still be `Scientific Format`.

![image](https://user-images.githubusercontent.com/8058899/139447992-a8952f0f-bf17-4a7e-9a57-eda7308022da.png)

Fixes #32710




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
